tests/sentry/lang/native/test_symbolicator.py

```diff
@@ -103,3 +110,129 @@ def test_sources_custom_disabled(default_project):
+
+    def test_notfound_deduplicated(self):
+        debug_id = "451a38b5-0679-79d2-0738-22a5ceb24c4b"
+        candidates = [
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path0",
+                "download": {"status": "notfound"},
+            },
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path1",
+                "download": {"status": "notfound"},
+            },
+        ]
+        response = {"modules": [{"debug_id": debug_id, "candidates": copy.copy(candidates)}]}
+        redacted = redact_internal_sources(response)
+        expected = [{"source": "sentry:microsoft", "download": {"status": "notfound"}}]
+        assert redacted["modules"][0]["candidates"] == expected
+
+    def test_notfound_omitted(self):
+        debug_id = "451a38b5-0679-79d2-0738-22a5ceb24c4b"
+        candidates = [
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path0",
+                "download": {"status": "notfound"},
+            },
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path1",
+                "download": {"status": "ok"},
+            },
+        ]
+        response = {"modules": [{"debug_id": debug_id, "candidates": copy.copy(candidates)}]}
+        redacted = redact_internal_sources(response)
+        expected = [{"source": "sentry:microsoft", "download": {"status": "ok"}}]
+        assert redacted["modules"][0]["candidates"] == expected
+
+    def test_multiple_notfound_filtered(self):
+        debug_id = "451a38b5-0679-79d2-0738-22a5ceb24c4b"
+        candidates = [
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path0",
+                "download": {"status": "notfound"},
+            },
+            {
+                "source": "sentry:microsoft",
+                "location": "http://microsoft.com/prefix/path1",
+                "download": {"status": "ok"},
+            },
+            {
+                "source": "sentry:apple",
+                "location": "http://microsoft.com/prefix/path0",
+                "download": {"status": "notfound"},
+            },
+            {
+                "source": "sentry:apple",
+                "location": "http://microsoft.com/prefix/path1",
+                "download": {"status": "ok"},
+            },
+        ]
+        response = {"modules": [{"debug_id": debug_id, "candidates": copy.copy(candidates)}]}
+        redacted = redact_internal_sources(response)
+        expected = [
+            {"source": "sentry:microsoft", "download": {"status": "ok"}},
+            {"source": "sentry:apple", "download": {"status": "ok"}},
+        ]
+        assert redacted["modules"][0]["candidates"] == expected
+
+    @pytest.mark.django_db
```

Comment:
which part requires this? test setup/teardown is 100x faster without and I skimmed over get_internal_source and didn't find anything

Reply:
Somehow `get_internal_sources` triggered needing this, I think it's reading the config options that wants all of the django init somehow.  But this has now changed to not need this anymore.